libc/src/stdio/generic/fileno.cpp

```diff
@@ -0,0 +1,20 @@
+//===-- Implementation of fileno
+//-------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "include/llvm-libc-types/FILE.h"
+#include "src/stdio/fileno.h"
+#include "src/__support/File/file.h"
```

Comment:
ah, sorry, my previous suggestion was bad. This should be:

```suggestion
#include "src/stdio/fileno.h"

#include "include/llvm-libc-types/FILE.h"
#include "src/__support/File/file.h"
```

We always want to include the .cpp's corresponding .h first, as then subsequent includes can sometimes be skipped by the preprocessor.